config/consumer/besluiten/delta-sync-dispatching.js

```diff
@@ -0,0 +1,80 @@
+const {
+  BYPASS_MU_AUTH_FOR_EXPENSIVE_QUERIES,
+  DIRECT_DATABASE_ENDPOINT,
+  PARALLEL_CALLS,
+  BATCH_SIZE,
+  SLEEP_BETWEEN_BATCHES,
+  INGEST_GRAPH,
+} = require("./config.js");
+const { batchedUpdate } = require("./utils");
+const endpoint = BYPASS_MU_AUTH_FOR_EXPENSIVE_QUERIES
+  ? DIRECT_DATABASE_ENDPOINT
+  : process.env.MU_SPARQL_ENDPOINT; //Defaults to mu-auth
+
+/**
+ * Dispatch the fetched information to a target graph.
+ * @param { mu, muAuthSudo, fetch } lib - The provided libraries from the host service.
+ * @param { termObjectChangeSets: { deletes, inserts } } data - The fetched changes sets, which objects of serialized Terms
+ *          [ {
+ *              graph: "<http://foo>",
+ *              subject: "<http://bar>",
+ *              predicate: "<http://baz>",
+ *              object: "<http://boom>^^<http://datatype>"
+ *            }
+ *         ]
+ * @return {void} Nothing
+ */
+async function dispatch(lib, data) {
+  const { mu, chunk } = lib;
+  const { termObjectChangeSets } = data;
+
+  const chunks = chunk(termObjectChangeSets, PARALLEL_CALLS);
+
+  for (const chunkedArray of chunks) {
+    await Promise.all(
```

Comment:
@nbittich could  catch the ones that error out and send this batches directly to virtuoso?
With the needed warning/errors etc etc

Reply:
@cecemel implemented in the latest  commit, please let me know if you think we should do something if it fails the second time